pkg/querier/worker/scheduler_processor_test.go

```diff
@@ -61,9 +61,11 @@ func TestSchedulerProcessor_processQueriesOnSingleStream(t *testing.T) {
 			case 1:
 				return &schedulerpb.SchedulerToQuerier{
 					QueryID: 1,
-					HttpRequest: &httpgrpc.HTTPRequest{
-						Method: "GET",
-						Url:    `/loki/api/v1/query_range?query={foo="bar"}&step=10&limit=200&direction=FORWARD`,
+					Request: &schedulerpb.SchedulerToQuerier_HttpRequest{
+						HttpRequest: &httpgrpc.HTTPRequest{
+							Method: "GET",
+							Url:    `/loki/api/v1/query_range?query={foo="bar"}&step=10&limit=200&direction=FORWARD`,
+						},
```

Comment:
should we have test(s) for the new field `QueryRequest` as well?

Reply:
Originally, I thought so too but then released that this is not really testing for the messages but rather the cancel behaviour.